server/src/assembly/resources/sbin/start-datanode.sh

```diff
@@ -97,6 +97,49 @@ launch_service()
 	return $?
 }
 
+
+# check whether tool 'lsof' & 'realpath' exists
+check_tool_env() {
+  if  ! type lsof > /dev/null 2>&1 ; then
+    echo ""
+    echo " Error: No tool 'lsof', Please install it."
+    exit 1
+  fi
+
+  if  ! type realpath > /dev/null 2>&1 ; then
```

Comment:
If don't have this `realpath` command on computer, then the IoTDB cannot start? It seems not friendly to users  who is tried to install IoTDB on edge devices...  As I know, even macos doesn't contain this command....

Reply:
Strange. 
1) In past start stop scripts, they also depend on some third-party tool  such netstat, ps etc.  If they are absent, user need also to insteall .
2) When I dev this PR, I specially test realpath in Linux and Mac. Normally, Linux and Mac has tool realpath. Does your Mac have no this tool ?
3) For edge device, if it is linux OS. Normally, it contain realpath. Because realpath belong to coreutils .

Reply:
Another possible solution is to gen 1 file in data directroy to save pid info.
Next time start-server.sh will use this file to check whether last process is still alive.

Reply:
Ok, it seems that the problem only appeared on Arm-base macos. What you mentioned does make sense.

One more suggestion, if users don't have these command on computer, I think we should skip these check and allow IoTDB to be started.

Reply:
I have just submit new codes. It does not use tool realpath now..